dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java

```diff
@@ -33,8 +33,12 @@
         this.text = text;
     }
 
-    public static DssRestContentType fromText(String text) {
-        return textMap.get(text);
+    public static DssRestContentType fromText(String text) {    	
+    	try{
+    		return textMap.get(text.split(";")[0]);
+    	}catch(NullPointerException e) {
+    		return textMap.get(text);
+    	}
```

Comment:
thanks to @Alencion's suggestion,
I think 

`
if (Objects.nonNull(text)){
        return textMap.get(text.split(";")[0]);
        }
        return null;
`
is easier to understand the code.

What do you think? @ztkmkoo

Reply:
I think check null looks better than catch the exception.

Reply:
I agree with you. I'll modify it.